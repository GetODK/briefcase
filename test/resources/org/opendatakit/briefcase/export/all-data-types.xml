<?xml version="1.0"?>
<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml">
  <h:head>
    <h:title>All data types</h:title>
    <model>
      <instance>
        <data id="all-data-types">
          <some_string/>
          <some_int/>
          <some_decimal/>
          <some_date/>
          <some_time/>
          <some_date_time/>
          <some_geopoint/>
          <some_geotrace/>
          <some_geoshape/>
          <some_barcode/>
          <meta>
            <instanceID/>
          </meta>
        </data>
      </instance>
      <bind nodeset="/data/some_string" type="string"/>
      <bind nodeset="/data/some_int" type="int"/>
      <bind nodeset="/data/some_decimal" type="decimal"/>
      <bind nodeset="/data/some_date" type="date"/>
      <bind nodeset="/data/some_time" type="time"/>
      <bind nodeset="/data/some_date_time" type="dateTime"/>
      <bind nodeset="/data/some_geopoint" type="geopoint"/>
      <bind nodeset="/data/some_geotrace" type="geotrace"/>
      <bind nodeset="/data/some_geoshape" type="geoshape"/>
      <bind nodeset="/data/some_barcode" type="barcode"/>
      <bind calculate="concat('uuid:', uuid())" nodeset="/data/meta/instanceID" readonly="true()" type="string"/>
    </model>
  </h:head>
  <h:body>
    <input ref="some_string"/>
    <input ref="some_int"/>
    <input ref="some_decimal"/>
    <input ref="some_date"/>
    <input ref="some_time"/>
    <input ref="some_date_time"/>
    <input ref="some_geopoint"/>
    <input ref="some_geotrace"/>
    <input ref="some_geoshape"/>
    <input ref="some_barcode"/>
  </h:body>
</h:html>