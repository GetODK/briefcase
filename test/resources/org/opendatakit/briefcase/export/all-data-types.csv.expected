SubmissionDate,some_string,some_int,some_decimal,some_date,some_time,some_date_time,some_geopoint-Latitude,some_geopoint-Longitude,some_geopoint-Altitude,some_geopoint-Accuracy,some_geotrace,some_geoshape,some_barcode,meta-instanceID,KEY
"Apr 26, 2018 8:58:20 AM",Hola,123,123.456,"Apr 26, 2018",8:56:00 AM,"Apr 26, 2018 8:56:00 AM",43.3149254,-1.9869671,71.80000305175781,15.478,43.314926 -1.9869713 71.80000305175781 10.0;43.3149258 -1.9869694 71.80000305175781 10.0;43.3149258 -1.9869694 71.80000305175781 10.0;,43.31513313655808 -1.9863833114504814 0.0 0.0;43.31552832470026 -1.987161487340927 0.0 0.0;43.315044828733015 -1.9877894595265388 0.0 0.0;43.31459255404834 -1.9869402050971987 0.0 0.0;43.31513313655808 -1.9863833114504814 0.0 0.0;,000049499094,uuid:39f3dd36-161e-45cb-a1a4-395831d253a7,uuid:39f3dd36-161e-45cb-a1a4-395831d253a7
